docs/bgp.md

```diff
@@ -0,0 +1,161 @@
+# Configuration of BGP peering
+
+By default, a Calico-Docker network uses a full BGP mesh to distribute routes 
+between all of the Calico nodes.  This approach has scale limitations and so 
+various different topologies are recommended for scaling out a network.
+
+This document describes the commands available in calicoctl for managing BGP 
+in a Calico-Docker networks.
+
+### Configuring the default node AS Number
+
+When creating a Calico node, you can optionally specify an AS number to use for
+the node.  If no AS Number if specified, the node will use the global default
+value.
+
+Use the "calicoctl bgp default-node-as" command to set the global default AS 
+Number.  If no value is configured, the default AS Number is 64511.
+
+The command does not need to be run on any specific node.
+
+For example, to set the default AS Number to 64512, enter the following 
+calicoctl command:
+
+	> calicoctl bgp default-node-as 64512
```

Comment:
Yup will change.  For the vagrant demo we display the full hostname which terminates with a $.